### ISSUE TITLE
Trend: Plot configuration discarded when change curves

### ISSUE THREAD

**Issue by tiagocoutinho** [taurus-org/taurus_pyqtgraph#74]
tested with:

- python 3.7
- pytango 9.3.1
- taurus 4.7.1-alpha
- taurus_pyqtgraph (latest develop from today)

I open the plot configuration panel where I have two curves.
I select curve A, change some parameters like curve color
I switch to curve B. When I change back to curve A the changes I made disappeared.

The workaround is to click apply before changing curve but it is counter intuitive and not very user friendly

Low priority for me

**Comment by cpascual** [comment]
Yes, I was aware of this but I hadn't created a ticket. Thanks @tiagocoutinho for doing it.

**Comment by tiagocoutinho** [comment]
Thanks for the quick reply @cpascual.
As I said before, not urgent and not blocking for me or any of my clients :-)

**Comment by cpascual** [comment]
Note: I changed from bug to enhancement because this is not a regression (it has always been like this in old qwt5 plots)
